Packs/MicrosoftADFS/ReleaseNotes/1_0_13.md

```diff
@@ -0,0 +1,4 @@
+
+#### Modeling Rules
+##### Microsoft ADFS Collection
+- Updated the Modeling Rules.
```

Comment:
please elaborate some more what the issue we fixed

Reply:
Done